tests/test-arg-parser.cpp

```diff
@@ -0,0 +1,95 @@
+#include <string>
+#include <vector>
+#include <sstream>
+
+#undef NDEBUG
+#include <cassert>
+
+#include "common.h"
+
+int main(void) {
+#ifdef _WIN32
+    printf("test-arg-parser: skip on windows build\n");
+#else
```

Comment:
What's the reason for skipping this on windows? A brief comment would be useful.

Reply:
Yeah I forgot to add a comment. Indeed, the test failed to compile because `setenv` is not supported on windows. I also exclude only the `setenv` part of the test: https://github.com/ggerganov/llama.cpp/pull/9308/commits/d545ffcb6deac469a0b4af0be69bf9b47997181e